src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int index = new Random().nextInt(RandomColor.values().length);
+        return RandomColor.values()[index].name();
+    }
+
+    public enum RandomColor {
+        RED,
+        ORANGE,
+        YELLOW,
+        GREEN,
+        BLUE,
+        PURPLE,
+        PINK,
+        GREY,
+        BLACK,
+        BROWN
+    }
```

Comment:
```suggestion
    public enum Color {
        RED,
        ORANGE,
        YELLOW,
        GREEN,
        BLUE,
        PURPLE,
        PINK,
        GREY,
        BLACK,
        BROWN
    }
```
This is enumeration of Colors, nothing said about random